packages/flutter/lib/src/services/keyboard_maps.dart

```diff
@@ -1493,3 +1493,230 @@ const Map<int, PhysicalKeyboardKey> kLinuxToPhysicalKey = <int, PhysicalKeyboard
+  77: LogicalKeyboardKey.keyM,
+  78: LogicalKeyboardKey.keyN,
+  79: LogicalKeyboardKey.keyO,
+  80: LogicalKeyboardKey.keyP,
+  81: LogicalKeyboardKey.keyQ,
+  82: LogicalKeyboardKey.keyR,
+  83: LogicalKeyboardKey.keyS,
+  84: LogicalKeyboardKey.keyT,
+  85: LogicalKeyboardKey.keyU,
+  86: LogicalKeyboardKey.keyV,
+  87: LogicalKeyboardKey.keyW,
+  88: LogicalKeyboardKey.keyX,
+  89: LogicalKeyboardKey.keyY,
+  90: LogicalKeyboardKey.keyZ,
+  49: LogicalKeyboardKey.digit1,
+  50: LogicalKeyboardKey.digit2,
+  51: LogicalKeyboardKey.digit3,
+  52: LogicalKeyboardKey.digit4,
+  53: LogicalKeyboardKey.digit5,
+  54: LogicalKeyboardKey.digit6,
+  55: LogicalKeyboardKey.digit7,
+  56: LogicalKeyboardKey.digit8,
+  57: LogicalKeyboardKey.digit9,
+  48: LogicalKeyboardKey.digit0,
+  13: LogicalKeyboardKey.enter,
+  27: LogicalKeyboardKey.escape,
+  8: LogicalKeyboardKey.backspace,
+  9: LogicalKeyboardKey.tab,
+  32: LogicalKeyboardKey.space,
+  189: LogicalKeyboardKey.minus,
+  187: LogicalKeyboardKey.equal,
+  219: LogicalKeyboardKey.bracketLeft,
+  221: LogicalKeyboardKey.bracketRight,
+  220: LogicalKeyboardKey.backslash,
+  186: LogicalKeyboardKey.semicolon,
+  222: LogicalKeyboardKey.quote,
+  192: LogicalKeyboardKey.backquote,
+  188: LogicalKeyboardKey.comma,
+  190: LogicalKeyboardKey.period,
+  191: LogicalKeyboardKey.slash,
+  20: LogicalKeyboardKey.capsLock,
+  112: LogicalKeyboardKey.f1,
+  113: LogicalKeyboardKey.f2,
+  114: LogicalKeyboardKey.f3,
+  115: LogicalKeyboardKey.f4,
+  116: LogicalKeyboardKey.f5,
+  117: LogicalKeyboardKey.f6,
+  118: LogicalKeyboardKey.f7,
+  119: LogicalKeyboardKey.f8,
+  120: LogicalKeyboardKey.f9,
+  121: LogicalKeyboardKey.f10,
+  122: LogicalKeyboardKey.f11,
+  123: LogicalKeyboardKey.f12,
+  145: LogicalKeyboardKey.scrollLock,
+  230: LogicalKeyboardKey.pause,
+  33: LogicalKeyboardKey.pageUp,
+  34: LogicalKeyboardKey.pageDown,
+  35: LogicalKeyboardKey.end,
+  36: LogicalKeyboardKey.home,
+  37: LogicalKeyboardKey.arrowLeft,
+  38: LogicalKeyboardKey.arrowUp,
+  39: LogicalKeyboardKey.arrowRight,
+  40: LogicalKeyboardKey.arrowDown,
+  45: LogicalKeyboardKey.insert,
+  46: LogicalKeyboardKey.delete,
+  144: LogicalKeyboardKey.numLock,
+  111: LogicalKeyboardKey.numpadDivide,
+  106: LogicalKeyboardKey.numpadMultiply,
+  109: LogicalKeyboardKey.numpadSubtract,
+  107: LogicalKeyboardKey.numpadAdd,
+  97: LogicalKeyboardKey.numpad1,
+  98: LogicalKeyboardKey.numpad2,
+  99: LogicalKeyboardKey.numpad3,
+  100: LogicalKeyboardKey.numpad4,
+  101: LogicalKeyboardKey.numpad5,
+  102: LogicalKeyboardKey.numpad6,
+  103: LogicalKeyboardKey.numpad7,
+  104: LogicalKeyboardKey.numpad8,
+  105: LogicalKeyboardKey.numpad9,
+  96: LogicalKeyboardKey.numpad0,
+  110: LogicalKeyboardKey.numpadDecimal,
+  93: LogicalKeyboardKey.contextMenu,
+  16: LogicalKeyboardKey.shift,
+  17: LogicalKeyboardKey.control,
+  18: LogicalKeyboardKey.alt,
+  91: LogicalKeyboardKey.meta,
+};
+
+// TODO(mdebbar): FINISH!
```

Comment:
Let's leave a link to an issue? https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-checking-in-comments-that-ask-questions

Reply:
Oops! This should be removed now.